airbyte-integrations/base/base.sh

```diff
@@ -14,7 +14,7 @@ function error() {
 # todo: make it easy to select source or destination and validate based on selection by adding an integration type env variable.
 function main() {
   CMD="$1"
-  shift 1
+  shift 1 || error "command not found."
```

Comment:
```suggestion
  shift 1 || error "command not specified."
```

Original sounds like it exists but doesn't belong to the set of allowed commands.

Reply:
done.